deepchem/models/torch_models/layers.py

```diff
@@ -4792,3 +4792,110 @@ def forward(self, dist: torch.Tensor) -> torch.Tensor:
         dist = dist.unsqueeze(-1) / self.cutoff
         output: torch.Tensor = self.envelope(dist) * (self.freq * dist).sin()
         return output
+
+
+class Highway(nn.Module):
+    """
+     Highway layer used in TextCNN model.
+     The Highway layer is a nn module that incorporates gating mechanisms to control
+     the flow of information between linear transformations with a non-linear activation function.
+     This enables better gradient propagation and information flow during training,
+     improving the performance of deep neural networks.
+     Custom initializers and activation functions can be easily integrated into this layer.
+     Examples
+     --------
+     >>> import torch
+     >>> from deepchem.models.torch_models import layers
+     >>> layer = layers.Highway(30, 2, 'kaiming_uniform_')
+     >>> input = torch.tensor([0, 1, 2, 3, 4, 5, 6, 7, 8, 9])
+     >>> output = layer(input)
+     >>> output.shape
+     torch.Size([5, 2])
+
+     """
+
+    def __init__(self,
+                 activation: str = 'relu',
+                 biases_initializer: str = 'constant_',
+                 weights_initializer='kaiming_uniform_',
+                 layer_shape=[5, 2],
+                 **kwargs):
+        super(Highway, self).__init__(**kwargs)
```

Comment:
`NameError: name 'init' is not defined` you cant pass arguments to .__init__()
Also docstrings are missing